frontend/src/hooks/useSessionValid.js

```diff
@@ -114,7 +114,8 @@ function useSessionValid() {
 
       if (err.request?.status === 412) {
         const domainName = JSON.parse(err.request.response).domain;
-        window.location.href = `/error?code=USF&domain=${domainName}`;
+        const code = JSON.parse(err.request.response).code;
```

Comment:
NIT:    better to do like this
``` js
    const response = JSON.parse(err.request.response);
    const domainName = response.domain;
    const code = response.code;
```

Reply:
@muhammad-ali-e  addressed your comments.